ogcapi-stable/ogcapi-collections/src/main/java/de/ii/ogcapi/collections/domain/CollectionsConfiguration.java

```diff
@@ -27,7 +56,15 @@
     abstract class Builder extends ExtensionConfiguration.Builder {
 
     }
-
+  /**
+   * @en Add additional links to the Collections resource. The value is an array of
```

Comment:
@CabukEge You also have to use the raw markdown here, so that for example "URI (\`href\`)" is preserved.
This applies to all files and methods.